modules/test/playwright/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+		"baseUrl": ".",
```

Comment:
What is this doing? :thinking:

Reply:
Submitted by error, forget about it...

Reply:
Sorry guys this is mine, I created a `tsconfg.json` with only this value for testing